lib/cryptoexchange/exchanges/gate/services/pairs.rb

```diff
@@ -0,0 +1,25 @@
+module Cryptoexchange::Exchanges
+  module Gate
+    module Services
+      class Pairs < Cryptoexchange::Services::Pairs
+        PAIRS_URL = "#{Cryptoexchange::Exchanges::Gate::Market::API_URL}/pairs"
+
+        def fetch
+          output = super
+          adapt(output)
+        end
+
+        def adapt(output)
+          output.map do |pair|
+            base, target = pair.split('_')
```

Comment:
Same as https://github.com/coingecko/cryptoexchange/pull/223/files#r152628195
`btc_usdt`, target should be BTC, base is USDT

Reply:
target/base correction